src/sonic-host-services/scripts/hostcfgd

```diff
@@ -8,8 +8,9 @@ import subprocess
 import syslog
 
 import jinja2
-from sonic_py_common import device_info
-from swsscommon.swsscommon import ConfigDBConnector
+from sonic_py_common.device_info import is_multi_npu, get_num_npus
+from swsscommon.swsscommon import SubscriberStateTable, DBConnector, Select
+from swsscommon.swsscommon import ConfigDBConnector, TableConsumable
```

Comment:
Do you still need ConfigDBConnector?

Reply:
Yes, i'm still using that for non-active listener use cases. The active listener responsibility is transferred to SubscriberStateTable

Reply:
What is the difference between active/non-active listeners?

Reply:
Sorry, i haven't framed it properly. I meant for some of the existing code which fetch table from the DB, i've still kept using ConfigDBConnector. Eg: https://github.com/vivekreddynv/sonic-buildimage/blob/1d626893843cd4514494f280fa7c1eb1159f002c/src/sonic-host-services/scripts/hostcfgd#L905